.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,38 @@
+name: pypi publishing
+
+
+on:
+  push:
+    tags: 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Just wondering whether there are any differences to what https://github.com/pypa/gh-action-pypi-publish propose in the README.

```
if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Reply:
Yes I also read that one. To be honest, I preferred the current version because I understood it a bit better.

The current version is inspired by [this SO comment](https://stackoverflow.com/questions/58862864/github-actions-ci-conditional-regex) and [NeuroR workflow](https://github.com/BlueBrain/NeuroR/blob/master/.github/workflows/upload-pypi.yml).